lib/matplotlib/pyplot.py

```diff
@@ -2434,627 +2424,613 @@ def spy(Z, precision=0, marker=None, markersize=None, aspect='equal', **kwargs):
 # to determine if they should trigger a draw.
 install_repl_displayhook()
 
-################# REMAINING CONTENT GENERATED BY boilerplate.py ##############
-
-
-# Autogenerated by boilerplate.py.  Do not edit as changes will be lost.
-@_autogen_docstring(Axes.acorr)
-def acorr(x, data=None, **kwargs):
-    ax = gca()
-    ret = ax.acorr(x, data=data, **kwargs)
-
-    return ret
-
-# Autogenerated by boilerplate.py.  Do not edit as changes will be lost.
-@_autogen_docstring(Axes.angle_spectrum)
-def angle_spectrum(x, Fs=None, Fc=None, window=None, pad_to=None, sides=None,
-                   data=None, **kwargs):
-    ax = gca()
-    ret = ax.angle_spectrum(x, Fs=Fs, Fc=Fc, window=window, pad_to=pad_to,
-                                sides=sides, data=data, **kwargs)
-
-    return ret
-
-# Autogenerated by boilerplate.py.  Do not edit as changes will be lost.
-@_autogen_docstring(Axes.arrow)
-def arrow(x, y, dx, dy, **kwargs):
-    ax = gca()
-    ret = ax.arrow(x, y, dx, dy, **kwargs)
 
-    return ret
-
-# Autogenerated by boilerplate.py.  Do not edit as changes will be lost.
-@_autogen_docstring(Axes.axhline)
-def axhline(y=0, xmin=0, xmax=1, **kwargs):
-    ax = gca()
-    ret = ax.axhline(y=y, xmin=xmin, xmax=xmax, **kwargs)
-
-    return ret
-
-# Autogenerated by boilerplate.py.  Do not edit as changes will be lost.
-@_autogen_docstring(Axes.axhspan)
-def axhspan(ymin, ymax, xmin=0, xmax=1, **kwargs):
-    ax = gca()
-    ret = ax.axhspan(ymin, ymax, xmin=xmin, xmax=xmax, **kwargs)
+################# REMAINING CONTENT GENERATED BY boilerplate.py ##############
 
-    return ret
 
 # Autogenerated by boilerplate.py.  Do not edit as changes will be lost.
-@_autogen_docstring(Axes.axvline)
-def axvline(x=0, ymin=0, ymax=1, **kwargs):
-    ax = gca()
-    ret = ax.axvline(x=x, ymin=ymin, ymax=ymax, **kwargs)
-
-    return ret
+@docstring.copy_dedent(Axes.acorr)
+def acorr(
```

Comment:
Is there a reason to put all the arguments on a new line. This looks rather unusual.

Reply:
If it's just because of the text wrapping, you could pass `initial_indent=' ' * (len(func) + 5)` to  `text_wrapper.fill()`. Then `lstrip()` or slice to `[len(func)+5:]` so that the first line has an appropriate length to fit after `def thefunc(`.

Reply:
It makes the autogeneration quite a bit simpler (specifically to properly wrap the first line -- the linewrapping code actually doesn't know ).
I would argue that while we want "reasonable" style for this autogenerated code, we shouldn't overthink it either...

Reply:
Then you end up with
```
def some_long_function(some_long_arg_list, ...
        some_more_args, etc)
```
which is not optimal either (you'd either want to align some_more_args under some_long_arg_list, or have a newline after the parenthesis).
The first solution is... honestly not worth it IMO.

Reply:
That's actually what I wanted. But not anymore after rechecking PEP-0008. :)

I'm fine with all args on new line for longer signatures. For short ones see my other comment.